modules/debugger.py

```diff
@@ -473,6 +479,17 @@ async def step_out(self) -> None:
 		try: await self.active.step_out(granularity=self.stepping_granularity())
 		except core.Error as e: self.console.error(f'Unable to step out: {e}')
 
+	@core.run
+	async def step_back(self) -> None:
+		try: await self.active.step_back(granularity=self.stepping_granularity())
+		except core.Error as e:
+			self.console.error(f'Unable to step back: Try running the "Debugger: Start recording for reverse debugging" command first. Details: {e}')
+
+	@core.run
+	async def record(self) -> None:
```

Comment:
Same as the `record` in session all this logic should be in the gdb adapter I think

Reply:
done